Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportRenderer.cpp

```diff
@@ -178,20 +178,13 @@ namespace EMStudio
         if (!m_actorEntities.empty())
         {
             // Find the actor instance and calculate the center from aabb.
-            AZ::Vector3 actorCenter = AZ::Vector3::CreateZero();
             EMotionFX::Integration::ActorComponent* actorComponent =
                 m_actorEntities[0]->FindComponent<EMotionFX::Integration::ActorComponent>();
             EMotionFX::ActorInstance* actorInstance = actorComponent->GetActorInstance();
             if (actorInstance)
             {
-                actorCenter += actorInstance->GetAabb().GetCenter();
+                result = actorInstance->GetAabb().GetCenter();
             }
```

Comment:
This should be the selected actor instance and not always the first one. Can be changed later but assuming this will lead to an issue sooner or later.

Reply:
The atom render viewport doesn't support selecting multiple actor yet. If we still planning to do so it will be an issue for sure.